noir/compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/big_int.rs

```diff
@@ -0,0 +1,54 @@
+use acvm::FieldElement;
+use num_bigint::BigUint;
+
+/// Represents a bigint value in the form (id, modulus) where
+///     id is the identifier of the big integer number, and
+///     modulus is the identifier of the big integer size
+#[derive(Default, Clone, Copy, Debug)]
+pub(crate) struct BigIntId(pub(crate) u32, pub(crate) u32); //bigint id, modulus id
```

Comment:
Lets use a structure with named fields instead of using a tuple and needing to remember which one of these is the bigint id versus the modulus id

Reply:
Done